pintail/site.py

```diff
@@ -1662,7 +1662,7 @@ def __init__(self, site, filename):
                                  stdout=subprocess.PIPE,
                                  env=site.get_script_env(self),
                                  universal_newlines=True)
-            self._configscr.readfp(p.stdout)
+            self._configscr.read(p.stdout)
```

Comment:
According to https://gitlab.gnome.org/Teams/Websites/help.gnome.org/-/blob/master/.gitlab-ci/fedora.Dockerfile#L35, this should work:

```suggestion
            self._configscr.read_file(p.stdout)
```

Reply:
True; read_file replaces readfp:
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read_file

Read would be for multiple file paths given:
https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.read

Reply:
I updated this PR, though I don't see a way forward for this abandoned library :(